### DISCUSSION TITLE
Problem with rofile.SetValue(ExifTag.ImageWidth, image.Width);

### DISCUSSION THREAD

**Discussion by zackmuc** [dlemstra/Magick.NET#1309] [Help]
I have the following code:

           using MagickImage image = new(tempImage);
            {
                var profile = new ExifProfile();
                profile.SetValue(ExifTag.Copyright, GetCopyright(count, country));
                profile.SetValue(ExifTag.ImageWidth, image.Width);
                profile.SetValue(ExifTag.ImageLength, image.Height);
}

it worked with MAgic:net 11.2.1 but with 12 i get 
Schweregrad	Code	Beschreibung	Projekt	Datei	Zeile	Unterdrückungszustand	Priorität
Fehler	CS0411	Die Typargumente der ExifProfile.SetValue<TValueType>(ExifTag<TValueType>, TValueType)-Methode können nicht per Rückschluss aus der Syntax abgeleitet werden. Geben Sie die Typargumente explizit an.	BingBackground	c:\Users\pauerbacher\Documents\Visual Studio 2022\Projects\BingBackground\Main\BingBackground\BingBackground.cs	337	Aktiv	Normal

It is an The type arguments for method 'method' cannot be inferred from the usage error.
Can anybody help?

Regards Peter

**Comment by dlemstra** [comment]
Implicit casts have been removed in version 12. You will need to do this instead:

```C#
profile.SetValue(ExifTag.ImageWidth, (Number)image.Width);
```

You can find a list of all changes here: https://github.com/dlemstra/Magick.NET/releases/tag/12.0.0

**Comment by zackmuc** [comment]
Thx 🥇
